components/eventing-controller/controllers/subscription/jetstream/reconciler_test.go

```diff
@@ -19,14 +19,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 
 	eventingv1alpha1 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha1"
+	jetstream2 "github.com/kyma-project/kyma/components/eventing-controller/controllers/subscription/jetstream"
```

Comment:
(nit) can we use better name than `jetstream2`?

Reply:
done